principles_guidelines.md

```diff
@@ -0,0 +1,124 @@
+
+
+<a name="mf"></a>
+##Mobile first
+
+**The site will be designed for smallest screens first**
+
+This is both a design ethos and a technical implementation strategy.
+
+Among other benefits, designing for smaller screens first helps to place focus on the core experience (which feeds into to the 'cutting the mustard' technique alluded to above).
+
+In terms of implementation, it dictates a strategy for the use of media queries.
+
+[back to top](#index)
+
+##Pattern Library
+
+**The site will contain a living pattern library**
+
+All components will be displayed in a pattern library, which shares it's code with the main website, ensuring it stays up to date.
+
+A pattern library will help us to deliver a consistent user experience and promote the reuse of code. It also provides other benefits such as simplifying CSS regression testing.
+
+
+<a name="performance"></a>
+## Performance
+
+[back to top](#index)
+
+**The site will deliver core content as fast as possible**  - (need to determine exactly what this means - working on separate doc)
+
+Performance needs to be a consideration throughout the build and we should always be mindful of limited hardware, poor latency, and low bandwidth situations.
+
+A performance budget should be used against which decesions about design, development, content, or any aspect of a site that may affect performance can be made. 
+
+
+###Performance budget
+
+- in progress...
+
+<a name="accessibility"></a>
+## Accessibility
+
+[back to top](#index)
+
+** The website will conform to WCAG 2.0 - level Double-A **
+
+The aim is to make the website accessible to the widest possible audience, regardless of the technology used to view it or the abilities of users. To this end, we aim to meet at least level Double-A conformance of the [Web Content Accessibility Guidelines (WCAG) 2.0](http://www.w3.org/TR/WCAG20/).
+
+The only exceptions to this will be where compliance is technically unfeasible and practical alternatives can be provided.  
+
+In addition, although reliance on client side scripts are permitted in the guidelines, we will strive to ensure that core functionality and information is available without them, in keeping with the [commitment to progressive enhancement](#pe), outlined above.
+
+<a name="support"></a>
+##Browser/device support
+
+[back to top](#index)
+
+**We will manually test the site on the following browsers***
+
+| Browser                               | Support Level |
+|:-------------------------------------:|:-------------:|
+| Chrome (latest 2 versions)            | Full          |
+| Safari (latest 2 versions)            | Full          |
+| Firefox (latest 2 versions)           | Full          |
+| Edge (latest 2 versions)              | Full          |
+| Internet Explorer (latest 2 versions) | Full          |
+| Android Browser                       | Full          |
```

Comment:
Hmm, nailing down Android Browser versions doesn't seem to be that straightforward.
https://decadecity.net/blog/2013/11/21/android-browser-versions
http://slides.com/html5test/the-android-browser#/

Given that and the fact that Android Browser accounts for less than 1% of traffic to our sites, it may be worth removing from the lis. What do you think?